core/src/com/po/fuck/model/constants/PhysicsConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.po.fuck.model.constants;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import static com.po.fuck.model.constants.ConstantsLoader.loadProperties;
+import static com.po.fuck.model.constants.ConstantsLoader.loadInt;
+
+public final class PhysicsConstants extends BaseConstants {
+    private static final String PHYSICS_PROPERTIES_PATH = Paths.get(PROPERTIES_FOLDER, "/physics.properties").toString();
+    private static final Properties PHYSICS_PROPERTIES = loadProperties(PHYSICS_PROPERTIES_PATH);
```

Comment:
Can't we just use `PATH`/`PROPERTIES_PATH` and `PROPERTIES`, we are already inside of `PhysicsConstants`, so prefixing is unnecessary ?